### ISSUE TITLE
Re-run PSTR with real data

### ISSUE THREAD

**Issue by Tilmon** [2DegreesInvesting/tiltIndicator#365]
Re-run PSTR after [Adapt str_companies#356](https://github.com/2DegreesInvesting/tiltIndicator/issues/356) is done with the following data: 

- `pstr_companies`: modified str_companies file from [Adapt str_companies#356](https://github.com/2DegreesInvesting/tiltIndicator/issues/356). _Note:_ I suggest to from now on, call `str_companies ` `pstr_companies `as we use 2 different company datasets for pstr and istr. 
- `str_ipr_targets `from [here](https://drive.google.com/drive/u/0/folders/15AsbEhET2WvU0z2dFhyKfvZF9jQHPuvP)
- `str_weo_targets `from [here](https://drive.google.com/drive/u/0/folders/15AsbEhET2WvU0z2dFhyKfvZF9jQHPuvP)

**Comment by Tilmon** [comment]
Dear @maurolepore , I included a description of the datasets in the task description.

**Comment by maurolepore** [comment]
The outputs are now uploaded to our drive ([slack comment](https://2investinginitiative.slack.com/archives/C050VAQACC9/p1685627282116389?thread_ts=1685626314.192949&cid=C050VAQACC9)).

Note the code I used currently lives in a private repo under my personal account: maurolepore/pstr and @kalashsinghal and @Tilmon have access to it. However I think it might be most useful if I moved it to the 2dii organization.
